test/tests/error.js

```diff
@@ -72,13 +72,6 @@ describe('xcomponent error cases', () => {
         });
     });
 
-    it('should try to render a component with a specified element when iframe mode is not supported', done => {
```

Comment:
Would ideally like to keep this test for the above reason.

Reply:
I feel this highlights the issue I have with this function. If an component does not support `iframe`, but does support `popup`, why should an error be thrown if a context isn't specified? Should is not default to `popup` in that case?